src/openrct2/actions/RideCreateAction.cpp

```diff
@@ -134,7 +134,6 @@ GameActions::Result RideCreateAction::Execute() const
         return GameActions::Result(GameActions::Status::Unknown, STR_CANT_CREATE_NEW_RIDE_ATTRACTION, STR_UNKNOWN_OBJECT_TYPE);
     }
 
-    ride->id = rideIndex;
```

Comment:
Why's this one unnecessary?

Reply:
RideAllocateAtIndex assigns this already.